modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/BackingServiceConfigurator.java

```diff
@@ -132,31 +132,45 @@ protected RetryPolicy buildRetryPolicy() {
 
         if (checkers == null) {
             checkers = new HashSet<>();
-
-            for (String template : configurationGenerator.getTemplateList()) {
+            List<String> items = configurationGenerator.getTemplateList();
+            // Add backing without template
+            items.add("elasticsearch");
+            items.add("kafka");
+            for (String item : items) {
                 try {
-                    File templateDir = configurationGenerator.getTemplateDirectory(template);
-                    String classPath = getClasspathForTemplate(template);
+                    log.debug("checker: " + item);
+                    File templateDir = getTemplateDir(item);
+                    String classPath = getClasspathForTemplate(item);
                     String checkClass = configurationGenerator.getUserConfig()
-                                                              .getProperty(template + PARAM_CHECK_SUFFIX);
-
+                                                              .getProperty(item + PARAM_CHECK_SUFFIX);
                     Optional<URLClassLoader> ucl = getClassLoaderForTemplate(templateDir, classPath);
                     if (ucl.isPresent()) {
                         Class<?> klass = Class.forName(checkClass, true, ucl.get());
+                        if (log.isDebugEnabled()) {
+                            log.debug("Adding checker " + item + " with class path: "
+                                    + Arrays.toString(ucl.get().getURLs()));
+                        }
```

Comment:
What about replacing current logger by the Log4j2 one and using the lambda feature?

Reply:
good point added a commit for that